core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/integration_test.go

```diff
@@ -661,7 +661,13 @@ func setupDependencies(t *testing.T, db *sqlx.DB, backend *backends.SimulatedBac
 	pollerLggr := logger.TestLogger(t)
 	pollerLggr.SetLogLevel(zapcore.WarnLevel)
 	lorm := logpoller.NewORM(big.NewInt(1337), db, pollerLggr, pgtest.NewQConfig(false))
-	lp := logpoller.NewLogPoller(lorm, ethClient, pollerLggr, 100*time.Millisecond, false, 1, 2, 2, 1000, 0)
+	lpOpts := logpoller.Opts{
+		PollPeriod:        100 * time.Millisecond,
+		FinalityDepth:     1,
+		BackfillBatchSize: 2,
+		RpcBatchSize:      2,
```

Comment:
Any reason for changing `KeepFinalizedBlocksDepth` value from `1000` to `0`?

Reply:
Nope... messed this up in lots of places, thanks